shells/flox-tests/default.nix

```diff
@@ -0,0 +1,51 @@
+{
+  self,
+  bats,
+  coreutils,
+  entr,
+  expect,
+  flox,
+  gawk,
+  git,
+  gnugrep,
+  gnused,
+  gnutar,
+  jq,
+  nix-editor,
+  parallel,
+  unixtools,
+  which,
+  mkShell,
+}: let
+  batsWith = bats.withLibraries (p: [
+    p.bats-assert
+    p.bats-file
+    p.bats-support
+  ]);
+in
+  mkShell {
+    name = "flox-tests";
+    buildInputs = [
+      batsWith
+      coreutils
+      entr
+      expect
+      flox
+      gawk
+      git
+      gnugrep
+      gnused
+      gnutar
+      jq
+      nix-editor
+      parallel
+      unixtools.util-linux
+      which
+    ];
```

Comment:
Why not just `inputsFrom = [ flox-tests ];`?

Reply:
Doesn't really work since I'm using `writeScriptBin`, but we don't need the dev shell anyway.